pkg/api/core/v1/client/apps.go

```diff
@@ -634,7 +640,8 @@ func (c *Client) AppPortForward(namespace string, appName, instance string, opts
 		PingPeriod: time.Second * 5,
 	})
 
-	wrapper := transport.NewBasicAuthRoundTripper(c.Settings.User, c.Settings.Password, upgradeRoundTripper)
+	// TODO: does it work?
```

Comment:
It seems so. I tried:

```
epinio app exec myapp
$ nc -l 9000
```

and from another terminal:

```
epinio app port-forward 9000
```

Opening localhost:9000 works and it prints things in the app exec terminal

We can remove the TODO